src/main/java/com/gmail/service/UserServiceImpl.java

```diff
@@ -0,0 +1,52 @@
+package com.gmail.service;
+
+import com.gmail.repository.UserRepository;
+import com.gmail.exceptions.UserNotFoundException;
+import com.gmail.model.User;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.List;
+
+/**
+ * Created by Space on 09.05.2019.
+ */
+@Log4j
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class UserServiceImpl implements UserService {
+    private final UserRepository userRepository;
+
+    @Override
+    public void save(User user) {
+        userRepository.save(user);
+    }
+
+    @Override
+    public User getById(Long id) {
+        return userRepository.findById(id).orElse(new User());
```

Comment:
Better return null or throw an exception

Reply:
At a first commit was userRepository.findById(id).orThrow(UserNotFoundException::new)

Reply:
At the first commit, you throw an exception and catch it in the same method!!! And it was not a good idea. So I left you a comment. But you resolve it not into the right way.

Reply:
Please fix it